### PR TITLE
LRDOCS-4309 clay styles

### PR DESCRIPTION
**Pull request by mwilliams2014**

### CODE REVIEW

File: develop/tutorials/articles/210-front-end-taglibs/04-clay-taglibs/09-clay-management-toolbar.markdown [hunk 1]
```diff
@@ -11,25 +11,49 @@ requires.
...
 
-`searchContainerId`: The ID of the search container connected to the management toolbar 
+`searchContainerId`: The ID of the search container connected to the management 
+toolbar 
 
 `selectable`: Whether to include a checkbox in the management toolbar
 
+`totalItems`: The total number of items across pagination. This number displays 
+when one or multiple items are selected.  
+<!--
```
[the remark this answers is not in the document]
Comment (carloslancha) [reply develop/tutorials/articles/210-front-end-taglibs/04-clay-taglibs/09-clay-management-toolbar.markdown]: Hey, "Select all" link is not present anymore in the management toolbar, yep. `totalItems`  still represents the number of items in all the dataset, just confirmed with @victorvalle

File: develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown [hunk 2]
```diff
@@ -18,102 +19,73 @@ Creating filters and sorting results involves the following process:
...
+
+        @Override
+        protected String[] getOrderByKeys() {
+          return new String[] {"name", "items", "status"};
+        }
+
+Now that you've defined the filter keys, you can provide the logic to sort the 
+columns. 
 
 ## Setting the Sorting Order for Columns [](id=setting-the-sorting-order-for-columns)
```
Comment (carloslancha) [inline develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown]: All this stuff about Comparator is relevant for the management toolbar? I feel like is something related to the search container, not to the toolbar. I'd remove points 1 and 2.

File: develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown [hunk 3]
```diff
@@ -268,6 +237,9 @@ build your portlet URL:
       return portletURL;
     }
 
+Now that the portlet URL is updated, you can update the search container to 
+reflect the changes.
+
 ## Including the Selected Column and Sorting Order in the Search Container [](id=including-the-selected-column-and-sorting-order-in-the-search-container)
```
Comment (carloslancha) [inline develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown]: The same as commented above, I think this is not relevant for a management toolbar implementation.

File: develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown [hunk 4]
```diff
@@ -238,16 +210,13 @@ Follow these steps to provide the logic for sorting your columns:
...
         }
 
-5.  Open the JSP containing the Management Toolbar and set its `sortingURL` 
-    attribute to the `getSortingURL()` method you defined in the display 
-    context, and set its `sortingOrder` attribute to the `getOrderByType()` 
-    method:
-    
-        sortingOrder=
-        "<%= ddmFormAdminDisplayContext.getOrderByType() %>"
+5.  Open the JSP view that contains the Clay Management Toolbar and set its
```
Comment (carloslancha) [inline develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown]: I'd move this point to the end of the doc

File: develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown (hunk 2 above)
Comment (mwilliams2014) [reply develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown]: My thought was that the Comparator was needed as part of the filtering and sorting for the toolbar. Without it, the filtering and sorting wouldn't work, right?

Comment (carloslancha) [reply develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown]: I'm not really sure about that. @jbalsas could you please take a look?

Comment (jbalsas) [reply develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown]: Is it really needed? I know @carloslancha has applied several toolbars and I don't think he had to create one from scratch as this explains. If we often simply use an existing one, that's what should be covered here, and how to create your own comparator/sorter should be moved to some advanced sections somewhere... this looks like it should affect SearchContainer and not ManagementToolbar... 🤔

Comment (mwilliams2014) [reply develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown]: Ok. Will do. Thanks @jbalsas @carloslancha

File: develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown [hunk 5]
```diff
@@ -7,14 +7,8 @@ filters and sorting order can be included in the JSP, but that can crowd the JSP
...
 tutorial. A Display Context is a Java class that defines the configuration for 
-an app's UI elements. 
-
-Creating filters and sorting results involves the following process: 
-
-- Creating the Filters
-- Creating a Comparator class for each field you want to sort
-- Including the selected column and sorting order in the portlet URL
-- Including the selected column and sorting order in the search container
+an app's UI elements. This tutorial explains how to create this filters in a
```
Comment (mwilliams2014) [inline develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown]: @jbalsas @carloslancha I've simplified the tutorial down to what's in this commit. when you say use an existing comparator, where can I find those so I can list them in the tutorial for others to use? Or do you think these steps are all that is required? Thanks!

Comment (mwilliams2014) [reply develop/tutorials/articles/110-portlets/08-applying-clay-styles-to-your-app/04-clay-management-toolbar/02-filtering-and-sorting-items-with-the-management-toolbar.markdown]: Cleared this up with Chema. Thanks!

**Comment by mwilliams2014** [comment]
Sent https://github.com/sez11a/liferay-docs/pull/3285

**Pull request closed without merging**